lib/granite/form/model/associations/reflections/embeds_many.rb

```diff
@@ -5,7 +5,7 @@ module Associations
         module Reflections
           class EmbedsMany < EmbedsAny
             def self.build(target, generated_methods, name, options = {}, &block)
-              target.add_attribute(Granite::Form::Model::Attributes::Reflections::Base, name) if target < Granite::Form::Model::Attributes
+              target.add_attribute(Granite::Form::Model::Attributes::Reflections::Base, name, type: Object) if target < Granite::Form::Model::Attributes
```

Comment:
Since we always need `type_definition` now, we need to pass `type` for all attributes. In this case these attributes are not used essentially but are here for ActiveRecord compatibility sake.